test/integration/targets/nxos_bgp/tests/common/supp_fib.yaml

```diff
@@ -0,0 +1,105 @@
+---
+- debug: msg="START TRANSPORT:{{ connection.transport }} nxos_bgp parameter test"
+
+- name: "Disable feature BGP"
+  nxos_feature:
+    feature: bgp
+    state: disabled
+    provider: "{{ connection }}"
+  ignore_errors: yes
+
+- name: "Enable feature BGP"
+  nxos_feature:
+    feature: bgp
+    state: enabled
+    provider: "{{ connection }}"
+  ignore_errors: yes
+
+- block:
+  # this task will fail on n9k running I2
+  # due to no support
+  - name: "set bestpath limit"
+    nxos_bgp: &set1
+      asn: 65535
+      vrf: "{{ item }}"
+      timer_bestpath_limit: 255
+      provider: "{{ connection }}"
+    with_items: "{{ vrfs }}"
+    register: result
+
+  - assert: &true
+      that:
+        - "result.changed == true"
+
+  - name: "Check Idempotence"
+    nxos_bgp: *set1
+    with_items: "{{ vrfs }}"
+    register: result
+
+  - assert: &false
+      that:
+        - "result.changed == false"
+
+  - name: "reset bestpath limit"
+    nxos_bgp: &reset1
+      asn: 65535
+      vrf: "{{ item }}"
+      timer_bestpath_limit: default
+      provider: "{{ connection }}"
+    with_items: "{{ vrfs }}"
+    register: result
+
+  - assert: *true
+
+  - name: "Check Idempotence"
+    nxos_bgp: *reset1
+    with_items: "{{ vrfs }}"
+    register: result
+
+  - assert: *false
+
+  # this task will fail on n9k running I2 or I4
+  # due to no support
+  - name: "set suppress fib"
+    nxos_bgp: &set2
+      asn: 65535
+      suppress_fib_pending: false
+      provider: "{{ connection }}"
+    register: result
+    ignore_errors: yes
+
+  - assert: *true
```

Comment:
`ignore_errors: yes` should be here as well in case the task fails, the result won't be changed to `true`.

Reply:
I removed ignore_errors: yes from &set2 task, because it was a typo. Actually, this test (set_suppress_fib) will fail on few platforms and the rescue block will print the error for it. This is what is desired, it should not ignore the error. For other platforms, it passes so there is no problems there. Thanks for catching it. 

All other changes you suggested are taken care of.